plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -186,7 +187,7 @@ internal fun Package.toSpdxPackage(
         copyrightText = licenseInfoResolver.getSpdxCopyrightText(id),
         description = description,
         downloadLocation = when (type) {
-            SpdxPackageType.PROJECT -> SpdxConstants.NONE
+            SpdxPackageType.PROJECT -> homepageUrl.nullOrBlankToSpdxNone()
```

Comment:
Hmm, the homepage URL is not necessarily the download location. So I'd instead use the source artifact URL or the VCS location. As projects are usually unpublished, there usually also is no source artifact, which leaves us with the VCS location. So I'd use that and only fall back to the homepage URL, if at all. What do you think @kikofernandez?

Reply:
A nit: In the below `when`s, you're handling the `SpdxPackageType.PROJECT` last. So for consistency I'd also handle it last here.